App/Containers/PhotoViewerScreen.js

```diff
@@ -39,8 +38,8 @@ class PhotoViewerScreen extends React.PureComponent {
 
   sharePressed () {
     const page = this.refs.gallery.currentPage
-    const hash = this.props.imageData[page].hash
-    this.props.authorShare(hash)
+    const item = this.props.imageData[page]
+    this.props.authorShare(item.hash)
```

Comment:
here

Reply:
`authorShare` is different than `share` though no?

Reply:
just made them both take item and dispatch item.hash for consistency

Reply:
ah right, didn't notice the difference